tools/gen_openvino_dockerfile.py

```diff
@@ -138,16 +136,15 @@ def dockerfile_for_windows(output_file):
 ARG OPENVINO_BUILD_TYPE
 
 WORKDIR /workspace
-RUN curl -L https://storage.openvinotoolkit.org/repositories/openvino/packages/2023.3/windows/w_openvino_toolkit_windows_2023.3.0.13775.ceeafaf64f3_x86_64.zip --output ov.zip
+RUN curl -L https://storage.openvinotoolkit.org/repositories/openvino/packages/2024.0/windows/w_openvino_toolkit_windows_2024.0.0.14509.34caeefd078_x86_64.zip --output ov.zip
```

Comment:
I see that we are hard-coding the windows build instruction to pick 2024.0 OV package. How would this build with 2023.3 OV if  TRITON_BUILD_OPENVINO_VERSION is specified as 2023.3?

Reply:
@tanmayv25 that windows path indeed is not managed by build.py. It is also different from linux building as it doesn't build openvino from source but it is using prebuilt package. That was the initial configuration for the windows build. Do you have some suggestions how it should be changed to manage the openvino version from build.py?

Reply:
The main issue we are unable to move freely with the OV version is because the ORT also has a OV dependency. We encounter symbol collisions when we their is a mismatch between the two OV versions. OV EP in ORT doesn't usually support the latest and greatest of the OV versions we would like the backend to move to. 
If we can properly isolate the symbols between two OV libraries then we can move these versions independent of each other.

Reply:
My suggestion would be to use `%OPENVINO_VERSION%` in windows to support both 2023.3 as well as 2024.0.